kata-webhook/Dockerfile

```diff
@@ -1,14 +1,14 @@
 # Copyright (c) 2019 Intel Corporation
 #
 # SPDX-License-Identifier: Apache-2.0
-FROM golang:latest AS builder
+FROM golang:1.16-alpine3.14 AS builder
```

Comment:
why we have a specific version here? can that be handle into the versions.yaml ?

Reply:
The webhook app [builds on Golang 1.16](https://github.com/kata-containers/tests/blob/main/kata-webhook/go.mod).

I'm still a learner of Golang so I don't know the best way to handle it. We can stick with `latest` if you think it is the right thing. Wdyt @GabyCT ?